readme.md

```diff
@@ -204,6 +204,64 @@ $publisher->createEditionToken('admin@exmaple.com', 74244556);
 ```
 $publisher->sendTargetedNotification(7145646, 'title', 'message', ['admin@example.com'], [ 356h64gqh65h545vhj6, 4574655w5ujw65w5a5], 'HIGH', 'false');
 ```
+#### Get Categories
+```
+$publisher->getCategories();
+```
+#### Create Category
+```
+$publisher->createCategory([
+    'categoryTitle'     => 'How to build a time machine',
+    'code'              => 'adventure',
+    'containsAll'       => 'true',
+    'defaultCategory'   => 'false',
+    'ordering'          => '4',
+    'publicationNodeId' => '61',
+]);
+```
+#### Delete ALL Categories
+```
+$publisher->deleteCategories(124252);
```

Comment:
Maybe it's worth adding a note that this deletes all categories within the publication node (with the ID being given as the param)?